doc/quickstart/actinia_quickstart.rst

```diff
@@ -18,75 +19,7 @@ satellite images, raster and vector data.
 
 .. contents:: Contents
 
-
```

Comment:
I removed all this because I was trying to reduce the length of the Quickstart. I'm not sure if it is okay to remove.

Reply:
The quick start has sections, and when doing a workshop the tutor, based on the objective of the workshop, can skip a section, therefore, the contents give a quick jump to the sections.
My preference is to keep it.

(If we are standardizing, and you remove it to one, then it has to be removed to all. In a meeting mention about keeping or removing the contents index of the quickstart, If it chosen to remove it then that task can be done in one PR that covers all)

Reply:
Hi Vicky it wasn't the contents I removed. It was the "quick tour"